### DISCUSSION TITLE
Vaultwarden Alpine LXC doesn't work anymore

### DISCUSSION THREAD

**Discussion by derlbear** [tteck/Proxmox#1484] [Help Needed] [answered]
It seems that an LXC-Update crashed my Vaultwarden instance. The access.log in /var/log/vaultwarden/ tells me this:

[2023-06-13 20:37:40.140][vaultwarden][ERROR] Web vault is not found at '/var/lib/vaultwarden/web-vault'. To install it, please follow the steps in: 
[2023-06-13 20:37:40.140][vaultwarden][ERROR] https://github.com/dani-garcia/vaultwarden/wiki/Building-binary#install-the-web-vault
[2023-06-13 20:37:40.140][vaultwarden][ERROR] You can also set the environment variable 'WEB_VAULT_ENABLED=false' to disable it

Since I installed Vaultwarden via script, I have no idea what I have to do to make it run again.

How about you?

**Comment by tteck** [comment]
What command did you run to perform an update?

**Reply by derlbear** [reply]
I only use your LXC Updater script for my LXC Containers.
For Vaultwarden (Alpine) I copied "bash -c "$(wget -qO - https://github.com/tteck/Proxmox/raw/main/ct/alpine-vaultwarden.sh)"" into the shell of the vaultwarden lxc.

**Reply by tteck** [reply]
It seems that you may have executed a different command. The command you mentioned, only carries out an `apk update && apk upgrade` operation. Restore a backup.

**Comment by tteck** [comment, marked as answer]
I apologize for the confusion caused by my lack of awareness regarding the extensive changes that had taken place. In any case, the script is now operational. It would be advisable to create a fresh Alpine-Vaultwarden LXC.

**Reply by AlexGeek77** [reply]
Just confirming, if I want to update my current Vaultwarden LXC you recommend I start a new alpine based one and load the old database?

**Reply by tteck** [reply]
Yes, backup/export and restore/import in the Vaultwarden UI

**Reply by derlbear** [reply]
Thanks for commenting! I was really confused because I didn't execute another command. Only a backup helped.

**Comment by jayphizzle** [comment]
> Yes, backup/export and restore/import in the Vaultwarden UI

I have the same issue as derlbear, so no access to the webgui. Where can I found/access the data folder of the vaultwarden installation (@alpine)? thx

**Reply by tteck** [reply]
https://github.com/tteck/Proxmox/issues/1617#issuecomment-1637041686

**Reply by jayphizzle** [reply]
Found the solution:

https://github.com/tteck/Proxmox/pull/1488/commits/26c5796ecdf2b41b52edbf4e309fe0e8f045ace4

Open /etc/conf.d/vaultwarden and change the WEB_VAULT_FOLDER to `/usr/share/webapps/vaultwarden-web/`

Restart vaultwarden: `rc-service vaultwarden restart`

Done
